xautocfg.cpp

```diff
@@ -248,6 +278,22 @@ int main(int argc, char **argv) {
 		}
 	};
 
+	auto execute_script = [&](int deviceid, bool enabled) {
+		if (enabled && !cfg.scripts.on_enable.empty()) {
+			std::cout << "executing custom script for enabled device=" << deviceid << std::endl;
+			int ret = system(cfg.scripts.on_enable.c_str());
```

Comment:
don't you need information passed to the script? but we can improve this later (e.g. with env-vars)

Reply:
Yeah, let add that later. For the moment, we can live with the current implementation.